packages/components/src/FullscreenModal.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from "react";
+import * as ReactDom from "react-dom";
+import styled, { StyledComponentClass } from "styled-components";
+import { colors } from "./styleConstants";
+
+export const ModalWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background-color: rgba(0, 0, 0, 0.4);
+  z-index: 1;
+`;
+
+export const ModalInner = styled.div`
+  min-width: 250px;
+  background-color: ${colors.basic.WHITE};
+  border: 1px solid ${colors.accent.CIVIL_GRAY_4};
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+  font-family: "Libre Franklin", sans-serif;
+`;
+
+export interface FullScreenModalProps {
+  open: boolean;
+  dismissOnOutsideClick?: boolean;
+  className?: string;
+  handleClose(): void;
+}
+
+export class FullScreenModal extends React.Component<FullScreenModalProps> {
+  public el: HTMLDivElement | undefined;
+
+  constructor(props: FullScreenModalProps) {
+    super(props);
+    this.state = {};
+    if (typeof document !== "undefined") {
```

Comment:
why not `document !== undefined`?

Reply:
if document is not defined this would throw an error "document is not defined", whereas if you do `typeof document` it returns the string "undefined"